eng/mgmt/Directory.Build.Mgmt.props

```diff
@@ -1,11 +1,11 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<PkgVersion>0.10.0</PkgVersion>
+		<PkgVersion>0.12.0-dev.20190715.1</PkgVersion>
```

Comment:
We should be able to eliminate this. I don't think we really need condition on the Import below. msbuild will handle the case where the package already exists.

Reply:
Ok here why you need this condition, I thought of adding a comment, but I am glad you caught this and now I will add a comment so that it is clear.
When you execute tests for the build tools tasks, it will actually will not restore the package for you during test execution if the project needs to resolve the imports and hence your test will fail.
So in that case, you will have to provide a conditional import and prior to executing tests you will have to restore it manually, hence you need it.